File: Nimbus-Config.py
#!/usr/bin/env python

import sys
import os
import logging

class Nimbus_Config:
    
    def __init__(self,NIMBUS_HOME_DIR):
        
        self.files_to_be_modified = {'common':NIMBUS_HOME_DIR+'/services/etc/nimbus/workspace-service/other/common.conf',
                                     'network_public':NIMBUS_HOME_DIR+'services/etc/nimbus/workspace-service/network-pools/public'}
        
        print self.files_to_be_modified
        
        self.logger = logging.getLogger('nimbus-config')
        hdlr = logging.FileHandler(NIMBUS_HOME_DIR+'/nimbus-config.log')
        formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
        hdlr.setFormatter(formatter)
        self.logger.addHandler(hdlr)
        self.logger.setLevel(logging.DEBUG)
        self.logger.debug("Nimbus Config file")
        
        

if __name__ == "__main__":
    
    print "Nimbus Config script"

    NIMBUS_HOME_DIR = os.getcwd()
    
    nc = Nimbus_Config(NIMBUS_HOME_DIR)
    
    

File: Nimbus-Config.rb
#!/usr/bin/env ruby

require('yaml')

class Nimbus
  
  def initialize()
    @@nimbus_home = Dir.pwd
    @@network_file = "#@@nimbus_home/services/etc/nimbus/workspace-service/network-pools/public"
    
    config_file = YAML.load_file('nimbus-config.yaml')
    
    @@IPStart = config_file['IPStart']
    @@Range = config_file['Range']
    
   #puts @@network_file
  end
  
  def add_IPAddresses()
    
    if !File.file?( @@network_file )
      puts " File Doesn't exist"
      exit
    end
    
    aFile = File.new(@@network_file, "a")
    
    x = @@IPStart.match(/(\d+).(\d+).(\d+).(\d+)/)
    
  
    
    for i in (0..@@Range-1)
      
      sum = Integer(x[4]) + i 
      
      ip = x[1]+"."+x[2]+"."+x[3]+"."+String(sum)
      
      aFile.syswrite(ip+"\n")
      
      #puts x[1]+"."+x[2]+"."+x[3]+"."+String(sum)
      #aFile.syswrite(i)
    end
    
    
    
    
  end
  
end




##### This is main program ###################

nimbus = Nimbus.new
nimbus.add_IPAddresses


